<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-N103UG2" failures="0" tests="5" name="AllTestCases.StudentFeePayment" time="10.314" errors="1" timestamp="2020-12-06T11:17:43 CST" skipped="4">
  <testcase classname="AllTestCases.StudentFeePayment" name="setUp" time="10.314">
    <error message="Cannot invoke &quot;String.contains(java.lang.CharSequence)&quot; because &quot;toEscape&quot; is null" type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "String.contains(java.lang.CharSequence)" because "toEscape" is null
at org.openqa.selenium.support.ui.Quotes.escape(Quotes.java:42)
at org.openqa.selenium.support.ui.Select.selectByValue(Select.java:197)
at LoginPage.Login.selectCountry(Login.java:27)
at LoginPage.Login.performLogin(Login.java:54)
at AllTestCases.StudentFeePayment.setUp(StudentFeePayment.java:47)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:61)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:366)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:320)
at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:176)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:122)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- setUp -->
  <system-out/>
  <testcase classname="AllTestCases.StudentFeePayment" name="VerifyFeeCashPayment" time="0.000">
    <skipped/>
  </testcase> <!-- VerifyFeeCashPayment -->
  <system-out/>
  <testcase classname="AllTestCases.StudentFeePayment" name="VerifyFeeChequePayment" time="0.000">
    <skipped/>
  </testcase> <!-- VerifyFeeChequePayment -->
  <system-out/>
  <testcase classname="AllTestCases.StudentFeePayment" name="VerifyFeeOnlinePayment" time="0.000">
    <skipped/>
  </testcase> <!-- VerifyFeeOnlinePayment -->
  <system-out/>
  <testcase classname="AllTestCases.StudentFeePayment" name="VerifyFeePaymentbywallet" time="0.000">
    <skipped/>
  </testcase> <!-- VerifyFeePaymentbywallet -->
  <system-out/>
</testsuite> <!-- AllTestCases.StudentFeePayment -->
